### PR TITLE
Feature/styled components

### LINKED ISSUE #17: Remove empty files
zB.
- client/component/elements/Card/Card.css
- client/component/elements/Flashcard/Flashcard.css
- client/component/layouts/Content/Content.css

### LINKED ISSUE #20: Labels
es hat Label, h2 und p elemente. 
ist das so in ordnung, oder sollten wir die Label Komponente erweitern, dass die size und andere props wie bold etc mitgegeben werden kann?

### LINKED ISSUE #38: Layout.css

### PR DESCRIPTION
**Pull request by g0tcoffee**
Switched to Styled Components
- Removed empty files which resolves #17 
- Migrated Global Styles in App.css to a Theme which resolves #38 
- Created new folder Typography containing several text based Styled Components which resolves #20 
- Created Styled Component Container
- Removed unnecessary class names
- Replaced almost all html tags with styled components

I really hope I've understood the spirit of atomic design correctly (the element's subfolders are a bit bloated and the naming seems a bit unintuitive).

### CODE REVIEW

File: src/client/components/typography/headings/TableHeader1.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+const StyledTableHeader1 = styled.h3`
+  font-weight: bold;
+  background-color: #f0f0f0;
+  padding: 10px;
+  text-align: center;
+`
+
+function TableHeader1(props: Readonly<TableHeader1Props>) {
```
Comment (frickerg) [inline src/client/components/typography/headings/TableHeader1.tsx]: Deskriptivere Namen verwenden ohne Zahlen

File: src/client/components/typography/headings/TableHeader2.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  children: React.ReactNode;
+}
+
+const StyledTableHeader2 = styled.h3`
+  font-weight: bold;
+  padding-top: 10;
+  padding-bottom: 0;
+`
+
+function TableHeader2(props: Readonly<TableHeader2Props>) {
```
Comment (frickerg) [inline src/client/components/typography/headings/TableHeader2.tsx]: Deskriptivere Namen verwenden ohne Zahlen

File: src/client/components/typography/headings/Header2.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+type Header2Props = {
+  children: React.ReactNode;
+}
+
+const StyledHeader2 = styled.h2`
+  margin: 10px;
+`
+
+function Header2(props: Readonly<Header2Props>) {
```
Comment (frickerg) [inline src/client/components/typography/headings/Header2.tsx]: Deskriptivere Namen verwenden ohne Zahlen

File: src/client/components/typography/headings/Header1.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+type Header1Props = {
+  children: React.ReactNode;
+}
+
+const StyledHeader1 = styled.h1`
+  font-weight: 500;
+  font-size: 30px;
+`
+
+function Header1(props: Readonly<Header1Props>) {
```
Comment (frickerg) [inline src/client/components/typography/headings/Header1.tsx]: Deskriptivere Namen verwenden ohne Zahlen

File: src/client/components/layouts/GamePage/EndPage.tsx [hunk 5]
```diff
@@ -1,9 +1,14 @@
 // TODO man könnte Label anstelle von h2 und p verwenden
```
Comment (frickerg) [inline src/client/components/layouts/GamePage/EndPage.tsx:1]: Erledigt?

**Review by frickerg — CHANGES_REQUESTED** [review]

File: src/client/components/layouts/GamePage/EndPage.tsx (hunk 5 above)
Comment (g0tcoffee) [reply src/client/components/layouts/GamePage/EndPage.tsx:1]: Danke dir, ist mir durch die Lappen gegangen.
Header Component habe ich gelassen, da bringt Label keinen Mehrwert.
Paragraph wurde ausgetauscht gegen neues ScoreLabel.

File: src/client/components/typography/headings/TableHeader1.tsx (hunk 1 above)
Comment (g0tcoffee) [reply src/client/components/typography/headings/TableHeader1.tsx]: Neu - FilledTableHeader

File: src/client/components/typography/headings/TableHeader2.tsx (hunk 2 above)
Comment (g0tcoffee) [reply src/client/components/typography/headings/TableHeader2.tsx]: Neu - SimpleTableHeader

File: src/client/components/typography/headings/Header1.tsx (hunk 4 above)
Comment (g0tcoffee) [reply src/client/components/typography/headings/Header1.tsx]: Neu - TitleHeader

**Pull request merged**